### PR TITLE
feature: fetch config

### PR DESCRIPTION
**Pull request by ErinHales**
TODO:
- [x] Gnosis chain showing up?
     - SDK was overriding header (fixed)
- [x] Fix `startWith` error
     - Missing env variable for avalanche

### CODE REVIEW

File: src/utils/config.ts [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
 import { NetworkMetadata, NetworkMap, TokenMetadataMap } from '@/config/types'
 import AVAXIcon from '@/assets/token-logos/AVAX.png'
 
+export const config = await import('@nomad-xyz/sdk-bridge').then(
+  async ({ BridgeContext }) => {
+    const { conf } = await BridgeContext.fetch(
+      process.env.VUE_APP_NOMAD_ENVIRONMENT,
+      false
+    )
+    return conf
```
Comment (ErinHales) [inline src/utils/config.ts:11]: I would like to just import `nomad` and get `nomad.conf` but couldn't get it to work...

File: src/views/TransactionSearch.vue [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <template>
   <!-- search transaction -->
   <n-card
-    style="width: 400px; backgroundColor: #2f2f2f;"
+    class="search-card"
```
Comment (ErinHales) [inline src/views/TransactionSearch.vue:4]: Fun story, the lint changed the capitalization to `backgroundcolor` and broke it

File: vue.config.js [hunk 3]
```diff
@@ -48,12 +48,4 @@ module.exports = defineConfig({
       return args
     })
   },
-  devServer: {
-    proxy: {
-      '^/api': {
```
Comment (ErinHales) [inline vue.config.js:53]: don't need this anymore.  Was for testing db locally

File: src/utils/config.ts (hunk 1 above)
Comment (Imti) [reply src/utils/config.ts:11]: This is fine for now, can fix later

File: src/views/TransactionSearch.vue (hunk 2 above)
Comment (Imti) [reply src/views/TransactionSearch.vue:4]: Lol!

**Review by Imti — APPROVED** [review]
🍔

**Pull request merged**